### ISSUE TITLE
Colour contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#665]

**Comment by KjerstiSteien** [comment]
Venter med denne til vi har refaktorert portal.
